src/jsMain/kotlin/io/beatmaps/user/account.kt

```diff
@@ -99,12 +107,12 @@ class AccountComponent : RComponent<AccountComponentProps, AccountComponentState
                 }
             }
             div("invalid-feedback") {
-                val error = state.usernameErrors.firstOrNull()
-                if (error != null) {
+                val errors = state.usernameErrors + state.descriptionErrors
+                if (errors.isNotEmpty()) {
                     attrs.jsStyle {
                         display = "block"
                     }
-                    +error
+                    +errors.joinToString(" ")
```

Comment:
So, as far as I know this change doesn't do anything because I think the sum total of the two lists can still only ever be one error.
I think the correct behaviour if there were multiple would be to have them on separate lines (see upload/main.kt where we a div is created for each).

Reply:
Or actually as there are two different buttons and requests maybe actually don't have them be mutually exclusive?